README.rst

```diff
@@ -0,0 +1,13 @@
+=================
+Choosing Boxes 
+=================
```

Comment:
SUPER minor and definitely an inconsequential issue. Just feel like it's a bit odd to see the three extra equal signs above and below the title. (i.e. I expected to see the equal signs to fit the title more snugly)
```
==============
Choosing Boxes 
==============
```